src/modules/media/services/media.service.ts

```diff
@@ -211,14 +215,47 @@ export class MediaService {
 		return `${this.mediaServiceUrl}/${embedUrl}?token=${playerTicket.jwt}`;
 	}
 
+	public async getThumbnailUrl(id: string, referer: string): Promise<string> {
+		const thumbnailPath = trimStart(await this.getThumbnailPath(id), '/');
+
+		const data = {
+			app: 'OR-*',
+			client: '', // TODO: Wait for reply on ARC-536 and implement resolution
+			referer: referer || this.host,
+			maxage: this.ticketServiceMaxAge,
+		};
+		this.logger.log(data);
```

Comment:
add todo to remove this once it runs smoothly on prod or make it a .info() call instead of a .log()